src/main/java/net/mcreator/ui/workspace/breadcrumb/WorkspaceFolderBreadcrumb.java

```diff
@@ -39,11 +39,19 @@
 public class WorkspaceFolderBreadcrumb extends JPanel {
 
 	private final MCreator mcreator;
+	private final boolean alwaysSuggestChoose;
+	private FolderElement currentFolder;
+	private SelectionListener selectionListener;
 
 	public WorkspaceFolderBreadcrumb(MCreator mcreator) {
+		this(mcreator, false);
+	}
+
+	public WorkspaceFolderBreadcrumb(MCreator mcreator, boolean alwaysSuggestChoose) {

```

Comment:
This also lacks any explanation of the new parameter whose name is not really intuitive

Reply:
Would it be cleaner if this was a predicate rather than a boolean constant?

Reply:
I don't even understand what it does and did not want to spend too much time figuring. This is why I asked from some explanation / JavaDOC